app/assets/stylesheets/alchemy/buttons.scss

```diff
@@ -49,8 +48,9 @@ button, input[type="submit"], a.button, input.button {
 }
 
 .icon_button {
-  width: 21px;
-  height: 21px;
+  @include box-sizing(border-box);
+  width: 27px;
```

Comment:
Properties should be ordered height, line-height, margin, overflow, padding, text-align, width